docs/spelling_wordlist.txt

```diff
@@ -80,13 +88,16 @@ Quickstart
 Rackspace
 Redis
 redis
+repo
 resize
 routable
 roadmap
+sdb
 snapshotting
 stateful
 sysadmin
 syslog
+systemctl
```

Comment:
This file is an extension to the dictionary. Elsewhere we use double backticks (like ``` ``systemctl`` ``` to format these things like code and keep them out of the dictionary.

I think @davidoliverSP2 is working on a documentation style guide somewhere.